### ISSUE TITLE
Install for entire system

### ISSUE THREAD

**Issue by ivanlan9** [stefanoborini/stefanoborini.github.io#4]
Nice summary, thanks. However, I have apps that are system applications. I'm using setuptools (didn't know about Poetry, I'll look into it), and of course, it's unsatisfactory. Could you cover the scenario where installing as a user defaults to virtual environments, while installing as root means that the app gets installed for everyone (consider how system files such as mime.types could be modified, or cron jobs installed).

Thanks!

**Comment by jugmac00** [comment]
You probably want to use `pipx`.

**Comment by cs01** [comment]
Yes pipx is the right tool for this.  https://github.com/pipxproject/pipx

**Comment by stefanoborini** [comment]
Sorry, been busy. I will write about it this weekend.
